Source/MSBuildTools.Unity.NuGet/MSBuildForUnity.csproj

```diff
@@ -27,13 +27,20 @@
     <!-- Major is hard-coded and should be incremented on breaking changes -->
     <MajorVersion>0</MajorVersion>
 
-    <MinorVersion>1</MinorVersion>
+    <!-- Minor version is hard-coded and should be incremented on feature additions that aren't breaking changes.-->
+    <MinorVersion>8</MinorVersion>
 
-    <!-- Revision is defaulted to 0, but if running in the lab where BuildID is specified then default the revision to that value. -->
+    <!-- Revision (semver Patch) version is hard-coded and should be incremented when a bug fix is made.-->
     <RevisionVersion>0</RevisionVersion>
-    <RevisionVersion Condition="'$(BUILD_BUILDID)' != ''">$(BUILD_BUILDID)</RevisionVersion>
-
-    <PackageVersion>$(MajorVersion).$(MinorVersion).$(RevisionVersion)</PackageVersion>
+    
+    <Version>$(MajorVersion).$(MinorVersion).$(RevisionVersion)</Version>
+    <!-- Version is based on Major.Minor.Revision, but if running in the lab where BuildID is specified then default then we append that.. -->
+    <Version Condition="'$(BUILD_BUILDID)' != ''">$(Version)-$(BUILD_BUILDID)</Version>
+
```

Comment:
This implies that the rolling "official" lab builds will always generate pre-release packages. I don't think we want that to be the case.

Reply:
I was also unclear on how you would get an actual "release" build without the suffix.

Reply:
Hmm, I think we can check on whether another property is set